src/plugins/legacy.js

```diff
@@ -38,7 +38,7 @@ export type LegacyCommand = {
 }
 
 export function convertFromV5 (c: LegacyCommand) {
-  if (!c.topic) throw new Error('command has no topic')
+  if (c.topic == null) throw new Error('command has no topic')
```

Comment:
What are you ultimately trying to do here? This doesn't change the logic so far as I can tell.

```
$ node -e "let types = [undefined, null, 'string']; types.forEach(t => console.dir({[t]: !t ? true : false}))"
{ undefined: true }
{ null: true }
{ string: false }
$ master  node -e "let types = [undefined, null, 'string']; types.forEach(t => console.dir({[t]: t == null ? true : false})
)"
{ undefined: true }
{ null: true }
{ string: false }
```